lib/views/feedbacks_slack_view.ex

```diff
@@ -1,7 +1,17 @@
 defmodule Aprb.Views.FeedbacksSlackView do
+  alias Aprb.Service.SentimentAnalysisService
+  
+  def prefix(event) do
+    emoji = event["properties"]["message"]
+      |>SentimentAnalysisService.sentiment_score
+      |>SentimentAnalysisService.sentiment_face_emoji
```

Comment:
Thanks for that tip, caught an unused import that way 👍 

I think this might be ready to go now.